pkg/util/ipset/ipset.go

```diff
@@ -275,8 +284,7 @@ func (runner *runner) CreateSet(set *IPSet, ignoreExistErr bool) error {
 	set.setIPSetDefaults()
 
 	// Validate ipset before creating
-	valid := set.Validate()
-	if !valid {
+	if err := set.Validate(); err != nil {
 		return fmt.Errorf("error creating ipset since it's invalid")
```

Comment:
you mean, 
```go
	// Validate ipset before creating
	err := set.Validate()
	if err != nil {
		return fmt.Errorf("error creating ipset since it's invalid")
	}
```
Like this?

Reply:
got it